Jenkinsfile

```diff
@@ -0,0 +1,409 @@
+// TODO check build log for disconnect/reconnect
+
+def IS_VALIDATED_MERGE_BUILD = false
+def GIT_COMMIT
+
+def cleanup = { ->
+  archive 'reports/**/*'
+  sh 'rm -f .env'
+
+  if (IS_VALIDATED_MERGE_BUILD && currentBuild.result != 'SUCCESS') {
+    withCredentials([usernamePassword(
+      credentialsId: '386d3445-b855-40e4-999a-dc5801336a69',
```

Comment:
Is it ok to have credentials hard coded in here? Should we be using the credentials jenkins plugin (if we have it)?